app/notifications/validators.py

```diff
@@ -134,6 +130,18 @@ def validate_and_format_recipient(send_to, key_type, service, notification_type,
         return validate_and_format_email_address(email_address=send_to)
 
 
+def check_if_service_can_send_to_number(service, number):
+    international_phone_info = get_international_phone_info(number)
+
+    if (
+        # if number is international and not a crown dependency
+        international_phone_info.international and international_phone_info.country_prefix != '44'
```

Comment:
I’m not too keen on having to look at the country prefix here. Having to look at the underlying data suggests that the properties of `international_phone_info` aren’t named well any more.

If you re-wrote your comment as code it would be clearer:
```python
if international_phone_info.international and not international_phone_info.crown_depedency:
```

Even then I’m not sure that’s totally accurate because Jersey and Guernsey aren’t a different nation.

Alternatively this code could stay the same and the code which sends the message to the provider could have something like:
```python
if international_phone_info.send_using_international_provider:
```

What do you think?

Reply:
to this thread: we decided to ship the code as it is to fix the bug, and add crown_depedency field to international_phone_info object next week.